rclcpp/include/rclcpp/node.hpp

```diff
@@ -1078,16 +1078,19 @@ class Node : public std::enable_shared_from_this<Node>
    * the node name, node namespace, topic type, endpoint type, participant's GID and its QoS
    * profile.
    *
-   * "topic_name" may be a relative, private or fully qualified topic name.
-   * A relative or private topic will be expanded using this node's namespace and name, if
-   * the "no_mangle" param is set to false.
+   * When the `no_mangle` parameter is `true`, the provided `topic_name` should be a valid topic
+   * name for the middleware (useful when combining ROS with native middleware (e.g. DDS) apps).
+   * When the `no_mangle` parameter is `false`, the provided `topic_name` should follow
+   * ROS topic name conventions.
    *
-   * The queried topic name is not remapped.
+   * `topic_name` may be a relative, private, or fully qualified topic name.
+   * A relative or private topic will be expanded using this node's namespace and name,
+   * The queried `topic_name` is not remapped.
```

Comment:
nit: shouldn't start in caps

Reply:
Oops. There was supposed to be a period instead of a comma.